e2e/utils/waits-utils.ts

```diff
@@ -225,15 +225,16 @@ export async function waitForNumberElements(page: Page,
  */
 export async function waitForText(page: Page,
                                   textSubstr: string,
-                                  selector: {xpath?: string, css?: string} = {css: 'body'}): Promise<boolean> {
+                                  selector: {xpath?: string, css?: string} = {css: 'body'},
+                                  timeOut?: number): Promise<boolean> {
   if (selector.css !== undefined) {
     try {
          // wait for visible then compare texts
       await page.waitForSelector(selector.css, {visible: true});
       const jsHandle = await page.waitForFunction((css, expText) => {
         const element = document.querySelector(css);
         return element && element.textContent.includes(expText);
-      }, {}, selector.css, textSubstr);
+      }, {timeout: timeOut}, selector.css, textSubstr);
```

Comment:
Since timeOut is an optional argument should there be a check to see if it exist before using it.

Reply:
`page.waitForFunction` ignores undefined optional param.